libafl/src/events/mod.rs

```diff
@@ -377,29 +375,32 @@ where
             /*Event::Custom {
                 sender_id: _, /*custom_event} => custom_event.name()*/
             } => "todo",*/
+            Event::Stop => "Stop",
         }
     }
 
-    fn name_detailed(&self) -> String {
+    /// Event's corresponding name with additional info
+    fn name_detailed(&self) -> Cow<'static, str> {
         match self {
             Event::NewTestcase { input, .. } => {
-                format!("Testcase {}", input.generate_name(None))
+                Cow::Owned(format!("Testcase {}", input.generate_name(None)))
```

Comment:
stupid question, that name might be wrong, right?

Reply:
I don't know but since this is not really related to this PR, I would like to avoid fixing all small issues around the codebase in this one